app/assets/javascripts/short-form/components/lendingInstitution.js.coffee

```diff
@@ -0,0 +1,46 @@
+angular.module('dahlia.components')
+.component 'lendingInstitution',
+  bindings:
+    application: '<'
+    form: '<'
+  templateUrl: 'short-form/components/lending-institution.html'
+  controller:
+    ['ShortFormApplicationService', 'ShortFormHelperService', '$translate',
+    (ShortFormApplicationService, ShortFormHelperService, $translate) ->
+      ctrl = @
+      @lendingInstitutions = ShortFormApplicationService.lendingInstitutions
+      @selectedInstitution = ''
+      @officers = []
+
+      @$onInit = ->
+        if @application.lendingAgent
+          @selectedInstitution = _.findKey(@lendingInstitutions, (agents) ->
+            _.some(agents, {'Id': ctrl.application.lendingAgent})
+          )
+          @officers = @lendingInstitutions[@selectedInstitution]
+
+      @inputInvalid = (fieldName) ->
+        form = @form.applicationForm
+        return false unless form
+        field = form[fieldName]
+        if form && field
+          field.$invalid && (field.$touched || form.$submitted)
+        else
+          false
+
+      @lendingInstitutionsNames = ->
+        _.keys(@lendingInstitutions)
+
+      @onChangeLendingInstitution = ->
+        @application.lendingAgent = null
+        @officers = @lendingInstitutions[@selectedInstitution]
```

Comment:
I think we should settle on just using lending agents/ agents for all of the variable names instead of loan officer. My understanding is that we're going to update the content to avoid the work "loan officer" at some point too.

Reply:
Sure, that makes sense.